Naive bayes

b-sherman opened the pull request:
I just created a new branch for the updated naive bayes files since the old one is very far behind now. Included are the images, code, and module text.

b-sherman committed:
Image does not belong in supervised folder

b-sherman committed:
Image does not belong in supervised folder

b-sherman committed:
Image does not belong in supervised folder

b-sherman committed:
Image does not belong in supervised folder

b-sherman committed:
Image does not belong in supervised folder

b-sherman committed:
Image does not belong in supervised folder

b-sherman committed:
Image does not belong in supervised folder

b-sherman committed:
Image does not belong in supervised folder

b-sherman committed:
Image does not belong in supervised folder

b-sherman committed:
Adding image for Bayes' Theorem equation

b-sherman committed:
Added code for naive bayes module

b-sherman committed:
Adding text for naive bayes module

b-sherman committed:
Updated text

b-sherman committed:
Updated code links

b-sherman committed:
Deleting obsolete image

b-sherman committed:
Deleting obsolete image

b-sherman committed:
Deleting obsolete image

b-sherman committed:
Deleting obsolete image

b-sherman committed:
Deleting obsolete image

b-sherman committed:
Deleting obsolete image

b-sherman committed:
Deleting obsolete image

mergu commented on docs/source/content/supervised/bayes.rst:
```diff
@@ -0,0 +1,180 @@
+Gaussian Model (Continuous)
+===========================
+Gaussian models assume features follow a normal distribution. As far as you
+need to know, a normal distribution is just a specific type of distribution.
```
We should let the user know briefly what each of these categories are. For instance, a gaussian distribution is one where most of the values fall towards the mean (maybe have a picture too)

mergu reviewed (commented):
This is very well written! I just had one suggestion

b-sherman committed:
Updated naive bayes text

+Added a graphic of some sort for each of the methods
+Explained Gaussian distributions a little more

b-sherman committed:
Added bell curve image for bayes

b-sherman committed:
Added code to generate the bell curve image

b-sherman committed:
Updated bell curve code to use seaborn

b-sherman committed:
Updated bell curve image to use seaborn

b-sherman commented:
I tried to add some more graphics to break up the text and apply the review comments.

mergu reviewed (approved):
This looks great Brendan. Thanks for the work you put in
